README.md

```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
 oc new-app -f https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml
 ```
 
+### OKD (Openshift-origin 3.11)
+
+There are multiple approaches to setup 3scale APIcast on Openshift origin. The one with yaml file is the common approach to setup 3scale account. Another approach is the easiest one where you need to pull 3scale image on local docker repository, then create new application in OKD from docker image by passing required parameters. I have used this approach to setup 3scale APIcast on my all in one OKD cluster on AWS.
+
+- Pull the 3scale image to your local docker repository.
+
+docker pull registry.access.redhat.com/3scale-amp20/apicast-gateway:1.0
+
+In all cases, in order to setup 3scale you need to have 3scale account. Steps to create and configure 3scale account are given here (https://github.com/tnscorcoran/rhsso-3scale).
+
+- Once configured 3scale account, you need 'ACCESS_TOKEN' along with your 3scale account URL which is called `ADMIN_PORTAL_DOMAIN`, to create secret which in turn used by your application for authentication. Create secret as discussed in the above approach.
```

Comment:
_Create secret as discussed in the above approach._ --> Is this a prerequisite? If not, please reword it to remove the term "above"